js/CreateMandate.js

```diff
@@ -24,29 +25,25 @@ cj(document).ready(function() {
      * Utility function to set the date on the %^$W#$&$&% datepicker elements
      * PRs welcome :)
      **/
-    function sdd_setDate(fieldname, date) {
+    function sdd_setDate(fieldname, date, recalculate = true) {
```

Comment:
This `recalculate` param can be ignored, does not work. It was to avoid recursive issues, when `sdd_setDate` is called from `sdd_recalculate_fields`. It is not enough, because there is also an event listener for `change` on most of the date fields.

Reply:
Thanks a lot @mlutfy, we'll try to pick this up after the holidays.

Happy holidays and new year!